src/schemas/recording/v1/browser.ts

```diff
@@ -73,16 +73,24 @@ const FormSubmittedEventSchema = BrowserEventBaseSchema.extend({
   submitter: ElementSelectorSchema,
 })
 
-const AssertedTextEventSchema = BrowserEventBaseSchema.extend({
-  type: z.literal('asserted-text'),
-  tab: z.string(),
-  selector: ElementSelectorSchema,
+const TextAssertionSchema = z.object({
+  type: z.literal('text'),
   operation: z.object({
     type: z.literal('contains'),
     value: z.string(),
   }),
 })
 
+// Will eventually be a union of different assertion types
+const AssertionSchema = TextAssertionSchema
+
+const AssertEventSchema = BrowserEventBaseSchema.extend({
+  type: z.literal('assert'),
```

Comment:
I know it breaks the past tense rule of event names, but it just sounds better as an imperative.